scripts/parser-tests/flow/allowlist.txt

```diff
@@ -14,9 +15,23 @@ class_properties/migrated_0003.js
 class_properties/migrated_0008.js
 class_properties/migrated_0021.js
 class_properties/migrated_0026.js
+comment_interning/class_method.js
+comment_interning/class_property.js
+comment_interning/declare_export_declaration.js
+comment_interning/declare_interface.js
+comment_interning/declare_type_alias.js
+comment_interning/decorator.js
+comment_interning/function_declaration.js
+comment_interning/import.js
+comment_interning/interface.js
+comment_interning/object_type.js
+comment_interning/remove_type_trailing_comments.js
+comment_interning/super.js
+comment_interning/type_alias.js
 decorators/migrated_0003.js
 export_import_reserved_words/migrated_0003.js
 export_statements/export_trailing_comma.js
+for_await_loops/migrated_0000.js
```

Comment:
For reference, this file now parses successfully due to top level await, 
```js
for await (let x of e) {}
```
and we assume all the Flow parser tests are running with `module` source type.